tests/testthat/test-translate-teradata.r

```diff
@@ -0,0 +1,150 @@
+context("translate-teradata")
+
+# Teradata base_scalar conversions -----------------------------------------
+
+test_that("as.numeric() translated to NUMERIC ", {
+  expect_equivalent(
+    translate_sql(as.numeric(field_name), con = simulate_teradata()),
+    sql("CAST(`field_name` AS NUMERIC)")
+  )
+})
+
+test_that("as.double() translated to NUMERIC ", {
+  expect_equivalent(
+    translate_sql(as.double(field_name), con = simulate_teradata()),
+    sql("CAST(`field_name` AS NUMERIC)")
+  )
+})
+
+test_that("as.character() translate to VARCHAR(MAX) ", {
+  expect_equivalent(
+  translate_sql(as.character(field_name), con = simulate_teradata()),
+    sql("CAST(`field_name` AS VARCHAR(MAX))")
+  )
+})
+test_that("log10() translates to LOG ", {
```

Comment:
Now that we have `sql_log()` used in multiple places, there's no need to test for each individual driver that uses it.